### ISSUE TITLE
Focus state on oc-loader and oc-spinner

### ISSUE THREAD

**Issue by PVince81** [owncloud/owncloud-design-system#544]
@marcus-herrmann as discussed, for accessibility we need to have a way to set the focus on spinners and loaders programmatically.

See https://github.com/owncloud/owncloud-design-system/pull/543 for how I did it for the autocomplete field.

I guess we also need to change the divs to be actually tab-able ?

We can schedule this task later on

**Comment by marcus-herrmann** [comment]
@PVince81 To make a spinner that is a div tabable and to make sure that its accessible name will be read out we have to do the following

* adding `tabindex="-1"`  so it can be focussed via JavaScript but not will become part of the taborder (so you don't manually tab to the item by accident)
* adding an accessible name, in this case by `aria-label`, since we can be sure that this part of the component will be translated by the backend
* adding a `role="img"` since `aria-label`/`aria-labelledby` doesn't work (reliably) on items without a role or elements that have `role="presentation"` (like `<div>` and `<span>` do)

**Comment by pascalwengerter** [comment]
OcLoader has an aria-label, but isn't focus-able as of now (but that wasn't required in the a11y-Workshops so far)

OcSpinner looks like this `<span :class="$_ocSpinner_class" :aria-label="ariaLabel" tabindex="-1" role="img" />` and should fullfil the requirements

Closing this for now
